content/applications/finance/accounting/reporting/overview/tax_carryover.rst

```diff
@@ -0,0 +1,38 @@
+=============
+Tax carryover
+=============
+
+When performing tax reports, the :guilabel:`tax carryover` feature allows carrying amounts from one
+period to the other without creating new entries.
+
+It has been created to respond to legal requirements in some locations where amounts must be
+transferred from period to period (for example, because the total of the line is negative).
+
+The feature is available per default in countries where it is required, such as Belgium, France, or
+Italy. There is no specific configuration required.
```

Comment:
The feature is activated by default in countries where it is required, such as Belgium, France, and
Italy. There is no specific configuration required.